docs/source/API/core/MultiGPUSupport.rst

```diff
@@ -0,0 +1,153 @@
+.. role:: cppkokkos(code)
+    :language: cppkokkos
+
+Multi-GPU Support
+=================
+
+Kokkos has support for launching kernels on multiple GPU devices within a single node from host on a single MPI rank for the CUDA, HIP, and SYCL backends.
+
+Using this feature requires knowledge of backend specific API calls for creating non-default execution space instances. Once the execution space has been
+created, it can be used to create execution policies, allocate views, etc. all on the device chosen by the user.
+
+Constructing Execution Spaces
+-----------------------------
+
+CUDA
+~~~~
+
+For CUDA backend, the user creates a ``cudaStream_t`` object and passes it to the ``Kokkos::Cuda`` constructor.
+
+.. note:: The user is expected to manage the lifetime of the stream. Only after the execution space is destroyed
+          can the user safely destroy the stream.
```

Comment:
```suggestion
.. note:: The user is expected to manage the lifetime of the stream. The related execution space needs to be destroyed before destroying the stream.
```

Reply:
maybe we should just move the note into the general part since we repeat it otherwise, right?